packages/wdio-applitools-service/src/index.js

```diff
@@ -51,14 +51,18 @@ export default class ApplitoolsService {
             return this.eyes.check(title, Target.window())
         })
 
-        global.browser.addCommand('takeRegionSnapshot', (title, region) => {
+        global.browser.addCommand('takeRegionSnapshot', (title, region, frame) => {
             if (!title) {
                 throw new Error('A title for the Applitools snapshot is missing')
             }
             if (!region || region === null) {
                 throw new Error('A region for the Applitools snapshot is missing')
             }
-            return this.eyes.check(title, Target.region(region))
+            if (!frame) {
+                console.log(frame)
```

Comment:
`console.log` should be removed or if you want to log something actual meaningful use the wdio logger

Reply:
Done